Content.Shared/Mobs/Components/MobStateComponent.cs

```diff
@@ -17,14 +18,25 @@ namespace Content.Shared.Mobs.Components
     public sealed partial class MobStateComponent : Component
     {
         //default mobstate is always the lowest state level
-        [ViewVariables] public MobState CurrentState { get; set; } = MobState.Alive;
+        [DataField, ViewVariables]
+        public MobState CurrentState { get; set; } = MobState.Alive;
 
```

Comment:
Is this set during init, how is it currently being determined.

Reply:
I moved my the stuff away from MobState so I'm not changing this anymore